src/report-python/report/reportmodule.c

```diff
@@ -44,14 +44,14 @@ PyObject *ReportError;
 static PyMethodDef module_methods[] = {
     /* method_name, func, flags, doc_string */
     /* for include/report/dump_dir.h */
-    { "dd_opendir"                , p_dd_opendir              , METH_VARARGS },
-    { "dd_create"                 , p_dd_create               , METH_VARARGS },
-    { "delete_dump_dir"           , p_delete_dump_dir         , METH_VARARGS },
+    { "dd_opendir", p_dd_opendir, METH_VARARGS, "Opens the given path"},
+    { "dd_create", p_dd_create, METH_VARARGS, "Create a dump directory"},
+    { "delete_dump_dir", p_delete_dump_dir, METH_VARARGS, "Delete a dump directory" },
     /* for include/report/report.h */
-    { "report_problem_in_dir"     , p_report_problem_in_dir   , METH_VARARGS },
-    { "report_problem_in_memory"  , p_report_problem_in_memory, METH_VARARGS },
-    { "report_problem"            , p_report_problem          , METH_VARARGS },
-    { NULL }
+    { "report_problem_in_dir", p_report_problem_in_dir, METH_VARARGS, "Reports a problem stored in a directory. Allows to specify flags."},
```

Comment:
So it allows specifying flags… Is it a positional argument? A keyword argument? Reading this would rather make me go read the code and ignore the docstrings.